lib/css_parser/parser.rb

```diff
@@ -21,6 +21,7 @@ class Parser
 
     # Initial parsing
     RE_AT_IMPORT_RULE = /\@import\s*(?:url\s*)?(?:\()?(?:\s*)["']?([^'"\s\)]*)["']?\)?([\w\s\,^\]\(\))]*)\)?[;\n]?/
+    RE_AT_FONTFACE_RULE = /\@font-face\s*/i
```

Comment:
Better name please, I get this is a regex, but the name does not tell me that, specifically what is RE?